q2_dada2/_denoise.py

```diff
@@ -138,6 +151,18 @@ def denoise_single(demultiplexed_seqs: SingleLanePerSampleSingleEndFastqDirFmt,
         return _denoise_helper(biom_fp, hashed_feature_ids)
 
 
+def denoise_single(demultiplexed_seqs: SingleLanePerSampleSingleEndFastqDirFmt,
+                   trunc_len: int, trim_left: int=0, max_ee: float=2.0,
+                   trunc_q: int=2, chimera_method: str='consensus',
+                   min_fold_parent_over_abundance: float=1.0, n_threads: int=1,
+                   n_reads_learn: int=1000000, hashed_feature_ids: bool=True
+                   ) -> (biom.Table, DNAIterator):
+    return _denoise_single(demultiplexed_seqs, trunc_len, trim_left, max_ee,
+                           trunc_q, 0, chimera_method,
```

Comment:
This is a little nit-picky, but would you mind specifying these parameters as keyword arguments (e.g., `trunc_q=trunc_q, max_len=0, ...`). I like to do this when there are a lot of parameters for readability (e.g., helps to understand why one of these is always set to `0`), and to avoid a mix-up in the future if new parameters are added to the function that is being called. This would go for the two calls to `_denoise_single`.